services/metadata/frontend/project-page/ProjectWidget.svelte

```diff
@@ -37,6 +42,17 @@
     <a class="data" href={$projectMetadata?.project.secondaryURL.url} target=_>{truncateString($projectMetadata?.project.secondaryURL.text)}</a>
   {/if}
 
+  <!-- ARK URLs -->
+  <div class=label>
+    <span style="display:inline">
+      Permalink
+    </span>
+      <a class="data" href={getARK()} target=_>Project Permalink</a>
+        <button on:click={copyToClipboard} title="copy permalink to the clipboard">
+          <svg class="icon" fill="none" stroke="currentColor" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M8 16H6a2 2 0 01-2-2V6a2 2 0 012-2h8a2 2 0 012 2v2m-6 12h8a2 2 0 002-2v-8a2 2 0 00-2-2h-8a2 2 0 00-2 2v8a2 2 0 002 2z"></path></svg>
+        </button>
+  </div>
```

Comment:
two things I noticed here:
- in how to cite, the copy-to-clipboard button is after the label, not after the value... so my guess is they are positioned differently now.
- normally, the label is a `<div class=label>` and then the value is in a separate `<div class=data>`. Here you have everything in the label div. This might explain the issue you mentioned that it's always displayed bold.

Reply:
Thx for the input!
I have removed the old logo, cleaned up the global.css, and fixed the permalink.